.github/workflows/main-build.yml

```diff
@@ -61,6 +61,14 @@ jobs:
           java-version: 17
           distribution: temurin
 
+      - name: Cache local Maven repository
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
```

Comment:
we don't use `pom.xml`. shouldn't this be `.kts`?

Reply:
Took another go at the key hash. Opted for a much more targeted hash that included the patch file hash.